### PR TITLE
Add new config command to easily open devproxyrc.json

### PR DESCRIPTION
**Pull request by mwiedemeyer**
I would like to add a new command `config` which automatically opens the `devproxyrc.json` in the default editor.
This would make it a lot easier for new developers to find the file.
The [documentation here](https://learn.microsoft.com/en-us/microsoft-cloud/dev/dev-proxy/get-started?tabs=automated&pivots=client-operating-system-windows#update-the-urls-to-watch) does not mention how to find the file. Instead it only says:
> In the Dev Proxy installation folder, open devproxyrc.json in a text editor.

So with the new command it would be a lot easier, because the documentation can say:
> Run `devproxy config` to open devproxyrc.json in a text editor

### CODE REVIEW

**Comment by waldekmastykarz** [comment]
Thank you for this PR. It'll definitely make it easier to update the default config. We've been hesitant to add in the past because it depends on the developer's file association, but thinking about it some more, it's a fair game to assume that folks have file associations configured on their machine. We'll process it asap. Thanks again!

**Comment by mwiedemeyer** [comment]
Awesome, thank you.

File: dev-proxy/ProxyHost.cs [hunk 1]
```diff
@@ -4,7 +4,9 @@
 using Microsoft.DevProxy.Abstractions;
 using Microsoft.Extensions.Logging;
 using System.CommandLine;
+using System.Diagnostics;
 using System.Net;
+using System.Reflection;
```
Comment (waldekmastykarz) [inline dev-proxy/ProxyHost.cs]: We don't need this

**Review by waldekmastykarz — APPROVED** [review]
Works like a charm!

**Pull request merged**